NeoML/Python/neoml/Dnn/Activation.py

```diff
@@ -585,6 +585,19 @@ def __init__(self, input_layer, name=None):
         internal = PythonWrapper.GELU(str(name), layers[0], int(outputs[0]))
         super().__init__(internal)
 
+    @property
+    def calculation_mode(self) -> tp.Literal["precise", "fast_approximate"]:
+        """ 'precise' (calculate GELU using the error function) or 'fast_approximate' (using an approximation x * sigmoid(1.702x))
+        """
+        return self._calculation_mode
```

Comment:
Why do you make a separate `self._calculation_mode` attribute here? It duplicates the field in the CGELULayer. I recommend add `CPyGELULayer::GetCalculationMode`